terraform/testcases/ecshealthcheck/otconfig.tpl

```diff
@@ -0,0 +1,30 @@
+extensions:
+  health_check:
+  health_check/1:
+    endpoint: "localhost:13133"
+    path: "/health/status"
+    check_collector_pipeline:
+      enabled: true
+      interval: "5m"
+      exporter_failure_threshold: 5
```

Comment:
Why do we need multiple health check extensions? Especially since `healthcheck/1` isn't defined in the service.

Reply:
Thanks. I have updated the code now.